### PR TITLE
Implemented Rest Controller and Re written the SampleBatchEntrySetup Page and SampleBatchEntry in new UI

### PR DESCRIPTION
**Pull request by manishjha-04**
**>Solving issue #700** 

**Before:**
Screenshot 2024-03-30 051355

Screenshot 2024-04-13 212040

**After:**

**If patient info is not selected**

https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/128405481/43632315-7d85-48c1-ab80-b9e9ce3aa84d

**If Patient Info is selected**

https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/128405481/6e693969-c64b-43f8-a9aa-e7ddc0464787

**If Pre-Printed Method is selected**

https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/128405481/68804eba-cd5f-4628-8b1a-899656796d19

### CODE REVIEW

**Comment by manishjha-04** [comment]
Hey @mozzy11, please review and let me know if any changes are needed. Also, I am working on the sampleBatchEntry page where the POST request is being handled. I'll be raising the PR soon!

**Comment by mozzy11** [comment]
Thanks @manishjha-04 . 
Looks good so far

**Comment by manishjha-04** [comment]
Apologies for the delay @mozzy11 . I've been bit busy with my university exams. But will surely wrap up this PR by Sunday and will let you know when it's ready for review.

**Comment by mozzy11** [comment]
No worries @manishjha-04 . 
I understand that . Great work so far

**Comment by manishjha-04** [comment]
Hey @mozzy11, it's still a work in progress. Instead of making the post request and redirecting it to the old UI, I'm working on the new UI for SampleBatchEntry as well!

**Comment by manishjha-04** [comment]
Hey @mozzy11 , it's ready for review. Let me know if any changes are needed. 

Also, let me know if you want me to split this PR to make the review process easier

File: src/main/java/org/openelisglobal/sample/form/ProjectData.java [hunk 1]
```diff
@@ -485,13 +485,6 @@ public void setEIDSites(List eIDSites) {
         EIDSites = eIDSites;
     }
 
-    public String getEIDsiteName() {
```
Comment (mozzy11) [inline src/main/java/org/openelisglobal/sample/form/ProjectData.java:488]: @manishjha-04 wont removing these properties brak the OLD ui ??

**Comment by mozzy11** [comment]
Thanks @manishjha-04 for the great work ??
Did you cinfirm your changes here dont break the OLD ui in any way ??

File: src/main/java/org/openelisglobal/sample/form/ProjectData.java (hunk 1 above)
Comment (manishjha-04) [reply src/main/java/org/openelisglobal/sample/form/ProjectData.java:488]: Hey @mozzy11 , I've verified, and the update doesn't affect the old UI.
 Additionally, the properties I removed were duplicates. You can check in the codebase to see these duplications.

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**